spec/features/category_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
- [ ] For some reason, the following specs are failing when I run rspec on them. Kindly look into why this is the case.

![image](https://user-images.githubusercontent.com/86318284/219852575-f8deab4b-d4a8-4a52-965f-285e5d0f785a.png)